policy/api/v1/client/client_test.go

```diff
@@ -175,10 +175,10 @@ func TestHTTPClient_Authorized(t *testing.T) {
 		handler := http2.Handler{StatusCode: http.StatusOK}
 		_, client := testServerAndClient(t, &handler)
 
-		response, err := client.Authorized(ctx, "http://::1:1", request)
+		response, err := client.Authorized(ctx, "http://test.test", request)
```

Comment:
bad test, uses external resources and assumes test.test won't be available?

Better use the test server and an invalid (local) port?

Reply:
from wiki

> .test is a reserved top-level domain intended for usage in software testing. It is guaranteed to never be registered into the Internet.

an ipv6 address won't work on our CI. A `.local` takes a second to (not) resolve.